README.md

```diff
@@ -236,6 +236,19 @@ It will be listening in the given address.
 
 You can read more about this profiler and the available endpoints in the [pprof documentation](https://pkg.go.dev/net/http/pprof).
 
+## Metrics
+Package registry is instrumented to expose Prometheus metrics. These metrics are exposed under the `/metrics` endpoint.
+By default this endpoint runs `localhost:9000`. These metrics can be scraped like:
```

Comment:
So far APM and pprof are disabled by default, should we do the same with metrics? This would also help to avoid backwards compatibility issues.

Reply:
Ok, I'll keep the same approach to avoid any possible backwards compatibility issue.